common/lib/xmodule/xmodule/js/src/problem/edit.coffee

```diff
@@ -552,20 +537,20 @@ class @MarkdownEditingDescriptor extends XModule.Descriptor
       });
 
       // split scripts and preformatted sections, and wrap paragraphs
-      splits = xml.split(/(\<\/?(?:script|pre).*?\>)/g);
-      scriptFlag = false;
+      splits = xml.split(/(\<\/?(?:script|pre|label|description).*?\>)/g);
+      wrappable = true;
```

Comment:
Can you add a comment about what "wrappable" signifies? I'm having trouble figuring that out.